streams-contrib/streams-persist-graph/src/main/java/org/apache/streams/graph/GraphHttpPersistWriter.java

```diff
@@ -218,7 +217,7 @@ protected ObjectNode executePost(HttpPost httpPost) {
     @Override
     public void prepare(Object configurationObject) {
```

Comment:
This method effectively ignores the `configurationObject` parameter -- should the parameter be removed?